Update troubleshooting.md (#23989)
DataDog/documentation: content/en/serverless/step_functions/troubleshooting.md

Please enable the `Include execution data` option on the state machine's logging. After enabling this option, log execution input, data passed between states, and execution output is logged. The Datadog backend uses the logs to construct these step spans for you.

## Some step spans are missing in the traces

> - Actions from Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported. `Activity` is not supported.

Reviewer: ```suggestion
- Actions from Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported.
- AWS Step Functions activities are not supported.
```

> - Actions of Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported, but`Activity` has not been supported yet.

Reviewer: ```suggestion
- Actions from Lambda, DynamoDB, StepFunction, and most of the other AWS services are supported. `Activity` is not supported.
```
Can you give me more context about what `Activity` means in this context?

Author: @urseberry Here is the [AWS's doc about StepFunction activity](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-activities.html).
